internal/provider/data_source_gitlab_project_tags.go

```diff
@@ -0,0 +1,117 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mitchellh/hashstructure"
+	"github.com/xanzy/go-gitlab"
+)
+
+var _ = registerDataSource("gitlab_project_tags", func() *schema.Resource {
+	return &schema.Resource{
+		Description: `The ` + "`gitlab_project_tags`" + ` data source allows details of a project tags to be retrieved by some search criteria.
```

Comment:
```suggestion
		Description: `The ` + "`gitlab_project_tags`" + ` data source allows details of project tags to be retrieved by some search criteria.
```

For the future: I think it makes sense to apply the suggestions 1:1 ;)